stanford-earth/matson
# Earth 1327 ally issues profile

# READY FOR REVIEW

# Summary
- Replacing the definition list markup with header and unordered list to make it more a117 on the profile page.

# Needed By (Date)
- Next push

# Steps to Test

1. Navigate to /people/ines-azevedo
2. Inspect markup for the Email and Links section
3. Verify the markup uses an H2 and UL.
4. Compare to PROD.  They should look the same visually.

# Associated Issues and/or People
- JIRA ticket SE3:  EARTH-1327 (A11y issues with Profile markup)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
